submissions/ebojudahCherish/js/main.js

```diff
@@ -0,0 +1,135 @@
+// //target the form element and all input fields 
+let submitForm = document.querySelector('#validateForm'),
+    firstName = document.querySelector('#firstName'),
+    lastName = document.querySelector('#lastName'),
+    phoneNumber = document.querySelector('#phoneNumber'),
+    email = document.querySelector('#email'),
+    password = document.querySelector('#passWord');
+    confirmPassword = document.querySelector('#confirmPassword')
+
+// this function makes sure all required fields are filled
+function required(field, event) {
+    if (field.value == '') {
+        field.nextElementSibling.innerHTML = 'This is a required field!';
+        event.preventDefault();
+        return false;
+    } else {
+        field.nextElementSibling.innerHTML = '';
+        return true; 
+    }
+}
+
+// add EventListener
+function validateForm(e) {
+    required(firstName, e);
+    required(lastName, e);
+    required(phoneNumber, e);
+    required(email, e);
+    required(password, e);
+    required(confirmPassword, e);
+    firstNameValidation(e)
+    lastNameValidation(e)
+    emailFieldValidation(e);
+    passwordValidation(e);
+    confirmPassWordValidation(e);
+    phoneNumberValidation(e);
+    
+    return true;
+}
+
+// firstName field validation
```

Comment:
`firstNameValidation`, `lastNameValidation`, `emailFieldValidation`, `phoneNumberValidation` and `passwordValidation` function look exactly the same. Simply write just a single function for them and pass the things that made them look a bit different as a parameter